pipeline/tasks/io_tasks.py

```diff
@@ -115,10 +120,16 @@ def execute(self, repo_root, files_dict={}):
         repo_dir = os.path.join(repo_root, "googleapis")
         # Write/overwrite the additonal files into the repo_dir so that user
         # can include additional files which are not in the public repo.
+        total_bytes = 0
         for f, content in files_dict.iteritems():
+            total_bytes += os.path.getsize(f)
             filename = os.path.join(repo_dir, f)
             if not os.path.exists(os.path.dirname(filename)):
                 os.makedirs(os.path.dirname(filename))
             with open(filename, "w+") as text_file:
                 text_file.write(base64.b64decode(content))
+        if total_bytes > _ZOOKEEPER_NODE_DATA_SIZE_LIMIT:
+            raise ValueError('size of merged local_repo exceeds limit of {} '
+                             'bytes; reduce the size of local_repo'.format(
+                                 _ZOOKEEPER_NODE_DATA_SIZE_LIMIT))
```

Comment:
Also output the size of the "merged local_repo". It's helpful for users to know how much they overshot the limit.